### ISSUE TITLE
Compatibility symfony 2.3

### ISSUE THREAD

**Issue by gpeanodc** [Spomky-Labs/IpFilterBundle#5]
I know this isn't really an issue, but cause I don't know where to write this..I apologize.

So, I want to know if this bundle might work on symfony 2.3. I know it's written in the doc that is compatible with 2.4 but I can't update my symfony version.

Thx.

**Comment by Spomky** [comment]
Hi,

First of all, excuse me for the very lonnnngggg delay between your issue and my answer.

If you use the last dev-master release, you should be able to install this bundle on Symfony 2.3.
But tests still fail.
I will investigate and try to fix bugs.

**Comment by Spomky** [comment]
Should work fine now with the v2.0.x.

```sh
composer require "spomky-labs/ip-filter-bundle" "2.0.*@dev"
```

**Comment by Spomky** [comment]
Fixed with 26def36914bce1e82c56c876ea4cd75e45fafb1d
